test/JLM_Tests/testng.xml

```diff
@@ -56,6 +56,7 @@
 			<class name="org.openj9.test.java.lang.management.TestMisc" />
 			<class name="org.openj9.test.java.lang.management.TestManagementUtils" />
 			<class name="org.openj9.test.java.lang.management.TestLoggingMXBean" />
+			<class name="org.openj9.test.java.lang.management.TestOpenJ9DiagnosticsMXBean" />
```

Comment:
Sorry for late review. I had issue with my email filtering.

TestOpenJ9DiagnosticsMXBean has annotation `level.extended` and it is added into JLM_Tests_IBMinternal in testng.xml.
In playlist.xml, we only run `sanity` for JLM_Tests_IBMinternal, which means we will not run tests in TestOpenJ9DiagnosticsMXBean.